comp230_01_1607539.tex

```diff
@@ -0,0 +1,65 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage{url}
+\usepackage[autostyle]{csquotes}
+
+% Please include a clear, concise, and descriptive title
+\title{What are the risks associated with nurturing a modding community, and how can game developers protect themselves?}
```

Comment:
I am left wondering to what extent the `"how can game developers protect themselves"` part of your question might be a step too far given the 1000-word constraint. There is little detailed legal discussion in your work that would answer the second half of your question. Consider culling this down to: `What are the risks and rewards associated with nurturing a modding community?`.